### PR TITLE
Moving 2 tests from KafkaMP to KafkaSE

### PR DESCRIPTION
**Pull request by jbescos**
This should solve the issues of that disabled 2 tests, because there is no other consumer in the moment we get the uncommitted events from Kafka.

### CODE REVIEW

**Comment by romain-grecourt** [comment]
@danielkec is this PR ready to be merged ?

File: tests/integration/kafka/src/test/java/io/helidon/messaging/connectors/kafka/KafkaSeTest.java [hunk 1]
```diff
@@ -334,4 +349,107 @@ void kafkaHeaderTest() throws InterruptedException {
...
+                        .valueDeserializer(StringDeserializer.class)
+                        .build()
+                )
+                .build();
+        List<String> uncommit = new ArrayList<>();
+        Channel6 kafkaConsumingBean = new Channel6();
+        Map<String, Object> config = new HashMap<>();
+        config.put("bootstrap.servers", KAFKA_SERVER);
+        config.put("key.serializer", LongSerializer.class.getName());
+        config.put("value.serializer", StringSerializer.class.getName());
```
Comment (danielkec) [inline tests/integration/kafka/src/test/java/io/helidon/messaging/connectors/kafka/KafkaSeTest.java:376]: This config seems not to be used anywhere

File: tests/integration/kafka/src/test/java/io/helidon/messaging/connectors/kafka/KafkaSeTest.java [hunk 2]
```diff
@@ -334,4 +349,107 @@ void kafkaHeaderTest() throws InterruptedException {
...
+                        .valueDeserializer(StringDeserializer.class)
+                        .build()
+                )
+                .build();
+        // Send the message that will not ACK. This will make in one partition to not commit any new message
+        Channel8 kafkaConsumingBean = new Channel8();
+        Map<String, Object> config = new HashMap<>();
+        config.put("bootstrap.servers", KAFKA_SERVER);
+        config.put("key.serializer", LongSerializer.class.getName());
+        config.put("value.serializer", StringSerializer.class.getName());
```
Comment (danielkec) [inline tests/integration/kafka/src/test/java/io/helidon/messaging/connectors/kafka/KafkaSeTest.java:431]: here too

**Comment by danielkec** [comment]
I have ran it 1564 times locally and it seems super stable 👍

File: tests/integration/kafka/src/test/java/io/helidon/messaging/connectors/kafka/KafkaSeTest.java (hunk 1 above)
Comment (jbescos) [reply tests/integration/kafka/src/test/java/io/helidon/messaging/connectors/kafka/KafkaSeTest.java]: I forgot to remove it. Now that config is in AbstractKafkaTest.produceAndCheck

**Pull request merged**